universal-application-tool-0.0.1/app/repository/ApplicantRepository.java

```diff
@@ -34,6 +39,32 @@ public ApplicantRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext exe
         () -> ebeanServer.find(Applicant.class).setId(id).findOneOrEmpty(), executionContext);
   }
 
+  public CompletionStage<ImmutableList<ProgramDefinition>> programsForApplicant(long id) {
```

Comment:
(nit): can you name the arg `applicantId` so it is really clear what the id is?

Reply:
Done!